src/NuGet.Clients/NuGet.SolutionRestoreManager/xlf/Resources.cs.xlf

```diff
@@ -24,7 +24,7 @@
       </trans-unit>
       <trans-unit id="InfoBar_HyperlinkGHCopilotDocs">
         <source>How to fix with GitHub Copilot</source>
-        <target state="translated">Postup opravy pomocí GitHub Copilotu</target>
+        <target state="new">How to fix with GitHub Copilot</target>
```

Comment:
I am confused. Not sure why we are updating the strings that are already localized.